src/usecases/createVisit.test.js

```diff
@@ -43,42 +29,40 @@ describe("createVisit", () => {
       callTime: date,
     };
 
-    const response = await createVisit(
-      randomIdProviderStub,
-      callIdProviderStub,
-      retrieveTrustByIdStub,
-      retrieveWardByIdStub,
-      createVisitUnitOfWorkSpy
-    )(visit, wardId, trustId);
+    const response = await createVisit(createVisitUnitOfWorkSpy)(
+      visit,
+      ward,
+      callId,
+      callPassword,
+      videoProvider
+    );
```

Comment:
this is why you are doing this. Feels much better now imo

Reply:
Not suggesting this for now ... but maybe it's possible to remove the partial application altogether here? Just pass the UOW as an argument?

Reply:
yeah good point. I remember you mentioning this last week too. This point makes me think about the way we are currying functions in general - as right now it seems there's a pattern of passing the function's dependencies through in the first argument set, treating that as a constructor, and then passing the function args through in second argument set. I suppose now that we are starting to move away from the AppContainer and are importing dependencies when we need them, do we still have the same need for curried functions? - e.g. in `book-a-visit` we now pass the unitOfWork to `createVisit` at the same time as we pass all the other args, so are in a place to be able to pass through all the args at the same time.